tests/post_training/model_scope.py

```diff
@@ -8,86 +8,65 @@
-            "quantization_params": {"preset": QuantizationPreset.MIXED, "model_type": ModelType.TRANSFORMER},
-        }
-    )
-    model_scope.append(
-        {
-            "name": "swin_base_patch4_window7_224",
-            "quantization_params": {"preset": QuantizationPreset.MIXED, "model_type": ModelType.TRANSFORMER},
-        }
-    )
-    # convit_tiny supressed due to bug - 104173
-    # model_scope.append({"name": "convit_tiny", "quantization_params": {"preset":QuantizationPreset.MIXED, "model_type": ModelType.TRANSFORMER}})
-    model_scope.append(
-        {
-            "name": "visformer_small",
-            "quantization_params": {"preset": QuantizationPreset.MIXED, "model_type": ModelType.TRANSFORMER},
-        }
-    )
-    model_scope.append(
-        {
-            "name": "crossvit_9_240",
-            "quantization_params": {"preset": QuantizationPreset.MIXED, "model_type": ModelType.TRANSFORMER},
-        }
-    )
-    # Others
-    model_scope.append({"name": "hrnet_w18", "quantization_params": {"preset": QuantizationPreset.MIXED}})
-    model_scope.append({"name": "efficientnet_lite0", "quantization_params": {"preset": QuantizationPreset.MIXED}})
-    model_scope.append({"name": "dpn68", "quantization_params": {"preset": QuantizationPreset.MIXED}})
-    model_scope.append({"name": "dla34", "quantization_params": {"preset": QuantizationPreset.MIXED}})
+def get_preset_enum_by_str(value: str) -> QuantizationPreset:
+    """
+    Return enum of QuantizationPreset by value.
+    """
+    for enum_value in QuantizationPreset:
+        if enum_value.value == value:
+            return enum_value
+    raise ValueError(f"QuantizationPreset does not contain {value}")
+
+
+def get_model_type_enum_by_str(value: str) -> ModelType:
+    """
+    Return enum of ModelType by value.
+    """
+    for enum_value in ModelType:
+        if enum_value.value == value:
+            return enum_value
+    raise ValueError(f"ModelType does not contain {value}")
+
+
+def get_validation_scope() -> dict:
+    """
+    Read json file that collected models to validation from MODELS_SCOPE_PATH.
+    Convert parameters
+
+    :return dict: _description_
+    """
+    model_scope = load_json(MODELS_SCOPE_PATH)
+
+    for model_name in list(model_scope.keys()):
+        model_info = model_scope[model_name]
+        if model_info.get("skipped"):
+            print(f"Skip {model_name} by '{model_info.get('skipped')}'")
+            model_scope.pop(model_name)
+            continue
+
+        qparams = model_info["quantization_params"]
+        if "preset" in qparams.keys():
+            qparams["preset"] = get_preset_enum_by_str(qparams["preset"])
+        if "model_type" in qparams.keys():
+            qparams["model_type"] = get_model_type_enum_by_str(qparams["model_type"])
 
     return model_scope
 
 
 VALIDATION_SCOPE = get_validation_scope()
+
+
+def get_cached_metric(report_model_name, metric_name):
+    cached_metric = -1
```

Comment:
Why not?
```python
cached_metric = NOT_AVAILABLE_MESSAGE
```

Reply:
Now all unavailable metrics saves as None instead of -1, and replaced to NOT_AVAILABLE_MESSAGE in report table.